### PR TITLE
docs(i18n): sync i18n files from Crowdin

### PR DESCRIPTION
**Pull request by guanbinrui**

### CODE REVIEW

File: src/page-components/PageFooter/useLocales/qya-AA.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Social": "crwdns9523:0crwdne9523:0",
```
Comment (UncleBill) [inline src/page-components/PageFooter/useLocales/qya-AA.json:2]: What does `crwdns9523:0crwdne9523:0` mean? These result might not suit for the `i18n` algorithm

File: src/pages/Edit/useLocales/ja-JP.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "Create maskbox": "MaskBoxを作成",
+  "Wallet account": "ウォレットアカウント",
+  "Collections": "コレクション",
+  "Total Amount": "合計金額",
+  "Creating...": "作成中…...",
```
Comment (guanbinrui) [inline src/pages/Edit/useLocales/ja-JP.json:6]:
```suggestion
  "Creating...": "作成中…",
```

File: src/pages/Edit/useLocales/ja-JP.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "Confirm": "確認",
+  "Description": "説明",
+  "MaskBox name": "MaskBox名",
+  "eg. Punk & Mask Special Edition": "例:Punk＆Mask特別版",
```
Comment (guanbinrui) [inline src/pages/Edit/useLocales/ja-JP.json:10]:
```suggestion
  "eg. Punk & Mask Special Edition": "例：Punk＆Mask特別版",
```

File: src/pages/Edit/useLocales/ja-JP.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  "Mystery thumbnail": "ミステリーサムネイル",
+  "Recommendation: 480*320/960*640": "推奨：480*320/960*640",
+  "Activity description": "アクティビティの詳細",
+  "eg.Rule Introduction": "例.ルールの紹介",
```
Comment (guanbinrui) [inline src/pages/Edit/useLocales/ja-JP.json:14]:
```suggestion
  "eg.Rule Introduction": "例：ルールの紹介",
```

**Pull request closed without merging**